docs/configuration.md

```diff
@@ -327,6 +327,16 @@ By default, the cache dynamically resizes according to the amount of free memory
 The option is set to 0 by default, which enables dynamic resizing. The default on previous versions of
 EventStoreDb was 100,000 entries.
 
+::: note
+The default value of 0 for `StreamInfoCacheCapacity` might not always be the best value for optimal performance. Ideally, it should be set to double the number of streams in the anticipated working set.
+
+The total number of streams can be obtained by checking the event count in the `$streams` system stream. This stream is created by the [$streams system projection](https://developers.eventstore.com/server/v24.2/projections.html#streams-projection).
+:::
+
+::: disclaimer
```

Comment:
I think that the disclaimer can be merged with the note above (unless `::: disclaimer` is actually a thing in the docs 😄 )
```suggestion
```

Reply:
Done